File: SDKs/amazonaws.com/codedeploy/2014-10-06/python/src/sdk/models/shared/listgithubaccounttokennamesoutput.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class ListGitHubAccountTokenNamesOutput:
    r"""ListGitHubAccountTokenNamesOutput
    Represents the output of a <code>ListGitHubAccountTokenNames</code> operation.
    """
    
    next_token: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('nextToken') }})
    token_name_list: Optional[List[str]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('tokenNameList') }})
    

File: SDKs/amazonaws.com/codedeploy/2014-10-06/typescript/src/sdk/models/shared/listgithubaccounttokennamesoutput.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ListGitHubAccountTokenNamesOutput
/** 
 * Represents the output of a <code>ListGitHubAccountTokenNames</code> operation.
**/
export class ListGitHubAccountTokenNamesOutput extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=nextToken" })
  nextToken?: string;

  @SpeakeasyMetadata({ data: "json, name=tokenNameList" })
  tokenNameList?: string[];
}
